CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.3.1 (Jul 15, 2015)
```

Comment:
Actually, this would be for 1.3.3 (sorry :/) ; Could you also specify that rsync_checkout_tag is now deprecated? Thanks again!